README.md

```diff
@@ -1,28 +1,78 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
-## Getting Started
+## Prerequisites
+
+The following tools are required to be installed:
+
+- [nvm](https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating) - Run the installation script: `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.1/install.sh | bash`
```

Comment:
This might be misleading and might cause confusion for users who want to use this repo. NVM should not be necessary to run this project, instead, the node version is the requisite. NVM is a tool to make it easier; not a requisite. 

Could this be edited so that it correctly matches its purpose?

Reply:
yes you are right, I included as a requirement and it is not :).

Thank you for pointing that out.